test-api-locally.md

```diff
@@ -1,21 +1,32 @@
 # Setup local Galasa APIs for testing
 
-### Overview
+## Overview
 
 In order to test new developments on the Galasa API servlets you need to be able to run them locally.
-The below instructions will help you setup couchDB and a local API server in order to run tests against the APIs.
+The below instructions will help you setup couchDB, etcd and a local API server in order to run tests against the corresponding galasa APIs.
 
-### 1. CouchDB
+## External Applications
+
+If you want to test the behaviour as if you are on an ecosystem you will need to install and run applications external to galasa.
```

Comment:
Maybe "If you want to test the API server as if it was running as part of an ecosystem..."